packages/gridjs/src/view/plugin/pagination/pagination.tsx

```diff
@@ -87,16 +88,18 @@ export class Pagination extends PluginBaseComponent<
       // we need to make sure that the state is set
       // to the default props when an error happens
       this.config.pipeline.on('error', () => {
+        console.error("An error ocurred.");
```

Comment:
redundant?

Reply:
Yes, I forgot to remove that line after testing.